src/components/Header/Nav/ProductNavigation/Desktop/ProductNavigation.tsx

```diff
@@ -45,31 +40,38 @@ export const ProductNavigation = ({ path, setNavMenuOpen, productsNav, subProduc
       <Root className={clsx(styles.root, styles.alignLeft)}>
         <List className={styles.list}>
           <Item>
-            <RadixTrigger className="nav-subproduct" ref={subProductMenuRef}>
-              <Trigger icon={icon} label={label} />
-            </RadixTrigger>
-            {(subProductTrigger || label === "Resources") && subProductsNav && (
-              <RadixContent className={styles.content}>
-                <SubProductContent subProductsNav={subProductsNav} />
-              </RadixContent>
-            )}
+            <span
```

Comment:
With your change this turned into a span. We need to ensure that it is correctly set for accessibility:
- Give it a role of button
- Set it an "aria-expanded" attribute
- Make sure the attribute is set to true when the dropdown opens